### DISCUSSION TITLE
soft & hard line breaks

### DISCUSSION THREAD

**Discussion by GambitFPS** [facebook/docusaurus#6528] [Q&A] [answered]
Is there a way to make soft line breaks in Docusaurus?
(in MarkText its `Shift` + `Enter` for example, but that doesnt seem to work in Docusaurus)

**Comment by Josh-Cena** [comment, marked as answer]
Docusaurus uses plain Markdown which only correlates character sequences with what's displayed, but <kbd>Shift</kbd> + <kbd>Enter</kbd> is a _gesture_, not a _character sequence_. In Markdown, you would typically use a `<br />` tag, or two trailing spaces followed by a line break to make a soft break.

I'm not very familiar with Mark text although I do have it installed, and I just tried to very surprisedly find out that it's not strictly Markdown, but a very weird combination of rich text and Markdown...
